src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,9 +1,50 @@
 import React, { Component } from 'react';
+import ProductImage from './components/ProductImage/ProductImage';
+import ProductInfo from './components/ProductInfo/ProductInfo';
+import ProductSelect from './components/ProductInfo/ProductSelect/ProductSelect';
 import './ProductDetail.scss';
 
 class ProductDetail extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productImageUrl: [
+        // 임시 값 -> mock 데이터로 변경 필요
+        '/images/productDetail/myImage.jpg',
+        '/images/productDetail/myImage2.jpg',
+        '/images/productDetail/myImage3.jpg',
+      ],
+    };
+  }
+
+  componentDidMount() {
+    // 상품 상세 페이지 패치해와야함
+  }
+
   render() {
-    return <div></div>;
+    const { productImageUrl } = this.state;
+
+    return (
+      <div className="productDetail">
+        <main>
+          <div className="path">Shop All {`>`}</div>
```

Comment:
```suggestion
      <main>
        <div className="path">Shop All {`>`}</div>
```
div 한 뎁스 사라져도 괜찮지 않을까요?